_posts/en/newsletters/2025-01-10-newsletter.md

```diff
@@ -0,0 +1,175 @@
+  results in it reserving 2,000 vbytes in block templates for coinbase
+  transactions rather than the intended 1,000 vbytes.  Each template
+  could include approximately five additional small transactions if the
+  double reservation was eliminated.  However, that could lead to miners
+  who depended on the double reservation producing invalid blocks,
+  resulting in a large loss of income.  Ismail analyzed past blocks to
+  attempt to determine which mining pools might be at risk.  He noted
+  Ocean.xyz and F2Pool are apparently using non-default settings; he also
+  found an unknown pool or miner creating large coinbase transactions
+  that could be a problem if the Bitcoin Core bug is fixed.  We ask
+  anyone who knows the unknown miner to alert them to the situation.
+
+  Jay Beddict [relayed][beddict double] the message to the Mining-Dev
+  mailing list.
+
+- **Contract-level relative timelocks:** Gregory Sanders
+  [posted][sanders clrt] to Delving Bitcoin about finding a solution for
+  a complication he discovered about a year ago (see [Newsletter
+  #284][news284 deltas]) when creating a proof-of-concept implementation
+  of [LN-Symmetry][topic eltoo].  In that protocol, each channel state
+  can be confirmed onchain but only the last state confirmed before a
+  deadline is able to spend the channel funds.  Usually, the parties to
+  a channel would only attempt to confirm the latest state; however, if
+  Alice initiates a new state update by partially signing a transaction
+  and sending it to Bob, only Bob can complete that transaction.  If Bob
+  stalls at that point, Alice can only close a channel in its
+  penultimate state.  If Bob waits until Alice's penultimate state has
+  almost reached its deadline and then gets the last state confirmed, it
+  will take approximately twice as long as the deadline for the channel
+  to resolve, called the (_2x delay problem_).  That means
+  [timelocks][topic timelocks] for [HTLCs][topic htlc] in LN-Symmetry
+  must be up to twice as long, which makes it easier for attackers to
+  prevent forwarding nodes from earning income on their capital (through
+  [channel jamming attacks][topic channel jamming attacks] and other
+  problems).
+
+  Sanders suggests solving the problem with a relative timelock that
+  would apply to all transactions required to settle a contract.  In
+  LN-Symmetry had such a feature and Alice got the penultimate state
+  confirmed, Bob would need to get the last state confirmed before the
+  deadline of the penultimate state.  In a [later post][sanders tpp],
+  Sanders links to a channel protocol by John Law (see [Newsletter
+  #244][news244 tpp]) that uses two transaction-level relative timelocks
+  to provide a contract-level relative timelock without consensus
+  changes.  However, that doesn't work for LN-Symmetry which allows each
+  state to spend from any previous state.
+
+  Sanders sketches a solution, but notes that it has problems.  He also
```

Comment:
perhaps s/problems/issues/ to avoid conflating with "the problem" in the next sentence

Reply:
Agree, or I'd prefer "drawbacks".